### ISSUE TITLE
Autoprefixer hangs on non existent filepath

### ISSUE THREAD

**Issue by Oikio** [nDmitry/grunt-autoprefixer#115]
Hello, I have this config:

```javscript
module.exports = function (ops) {
  return {
    options: {
      browsers: ['last 2 version', 'ie 9', 'opera 10']
    },
    main: {
      files: [
        {
          expand: true,
          cwd: '.tmp',
          src: ['styles/**/*.css'],
          dest: '.tmp'
        }
      ]
    },
    blocks: {
      files: [
        {
          expand: true,
          cwd: '.tmp',
          src: ['blocks/**/*.css'],
          dest: '.tmp'
        }
      ]
    }
  }
}
```
When blocks folder does not exist autoprefixer hangs on this task in 3.0.1, 3.0.0 is working fine.

**Comment by nDmitry** [comment]
Yeah I will come up with a fix for this soon. It is already deployed in [grunt-postcss](https://github.com/nDmitry/grunt-postcss), so check it out if it's urgent.

**Comment by Oikio** [comment]
@nDmitry Oh, great, will stay on 3.0.0 for a while. Thanks!

**Comment by 0xR** [comment]
I am running into this issue and there are more people who have this issue:
https://stackoverflow.com/questions/30604371/yoeman-task-autoprefixerdist-gets-stuck

**Comment by nDmitry** [comment]
Fixed in 3.0.2
